terraform/ingest/index.js

```diff
@@ -121,7 +121,9 @@ function processFiles () {
     // YYYYMMDD format
     let date = f.replace('.csv', '')
     // Since we end up sending this to the frontend, make it parseable upfront
-    date = `${date.slice(0, 4)}-${date.slice(4, 6)}-${date.slice(6, 8)}`
+    let re = /(\d{4})-*(\d{2})-*(\d{2})/
```

Comment:
Disagree on the regex.  We don't need to handle arbitrary dates, just the ones we're generated on L247.  Just take the first 10 characters of the string (`2018-03-02T11:22:19.514Z` -> `2018-03-02`).

I'll update all the input filenames on S3 to be the same ISO string format (e.g. `2018-03-02T11:22:19.514Z`).